Src/AutoFixture.NUnit3/InlineAutoDataAttribute.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework.Interfaces;
+
+namespace Ploeh.AutoFixture.NUnit3
+{
+    /// <summary>
+    /// This attribute acts as a TestCaseAttribute but allow incomplete parameter values, 
+    /// which will be provided by AutoFixture. 
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    [CLSCompliant(false)]
+    public class InlineAutoDataAttribute : AutoDataAttribute
```

Comment:
I gave this a second thought and you're right, it will be a breaking change.

(The reason is that the CompositeDataAttribute is a *DataAttribute*. I had in mind that it was an AutoDataAttribute...)